FhirToDataLake/synapse/DicomToSynapseValidator.py

```diff
@@ -0,0 +1,121 @@
+import os
+import sys
+import argparse
+import json
+import requests
+import pyodbc
+import pandas as pd
+import time
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--synapse_workspace", required=True, help="Name of Synapse workspace.")
+parser.add_argument("--dicom_server_url", required=True, help="Dicom server url.")
+parser.add_argument("--dicom_server_access_token", help="Dicom server bearer access token.")
+parser.add_argument("--schema_directory", required=True, help="Schema directory path.")
```

Comment:
Would it be convenient to add customized schema testing for DICOM as well?

Reply:
Added customized schema test.